### PR TITLE
added resource configurations for NCSA Delta

### PR DESCRIPTION
**Pull request by mtitov**

### CODE REVIEW

File: src/radical/pilot/configs/resource_ncsa.json [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+                                         "order": ["SRUN"],
+                                         "SRUN" : {}
+                                        },
+        "pre_bootstrap_0"             : [
+                                         "module reset",
+                                         "module load python"
+                                        ],
+        "default_remote_workdir"      : "/scratch/%(pd.project)s/$USER",
+        "python_dist"                 : "default",
+        "virtenv_mode"                : "create",
```
Comment (andre-merzky) [inline src/radical/pilot/configs/resource_ncsa.json:32]: would it make sense to default to the `local` `virtenv_mode`?

**Review by andre-merzky — APPROVED** [review]
Left a question, otherwise looks good to me.

File: src/radical/pilot/configs/resource_ncsa.json (hunk 1 above)
Comment (mtitov) [reply src/radical/pilot/configs/resource_ncsa.json:32]: yeah, will do

Comment (mtitov) [reply src/radical/pilot/configs/resource_ncsa.json]: done

**Pull request merged**